controllers/util/prometheus_exporter_util.go

```diff
@@ -103,7 +103,7 @@ func GenerateSolrPrometheusExporterDeployment(solrPrometheusExporter *solr.SolrP
 		exporterArgs = append(exporterArgs, "-z", solrConnectionInfo.CloudZkConnnectionInfo.ZkConnectionString())
 
 		// Add ACL information, if given, through Env Vars
-		if hasACLs, aclEnvs := AddACLsToEnv(solrConnectionInfo.CloudZkConnnectionInfo); hasACLs {
+		if hasACLs, aclEnvs := AddACLsToEnv(solrConnectionInfo.CloudZkConnnectionInfo.AllACL, solrConnectionInfo.CloudZkConnnectionInfo.ReadOnlyACL); hasACLs {
```

Comment:
Do we want to get this out of the SolrCloud object if they do a SolrCloudReference by name in the **same** namespace? Not sure we do, and could be a fix for the future if so.

Reply:
This is definitely a change to make separately. Since it has very little to do with the scope of this PR.

Reply:
Just so I'm clear ... no change needed here for this PR, correct?

Reply:
Upon further reflection, yes